tests/integration/test_run.py

```diff
@@ -84,17 +84,6 @@ def test_all_sets_and_all_seasons(self):
                 msg = "Count for {} is invalid: {}"
                 self.fail(msg.format(set_name, count))
 
-        # enso_diags and streamflow only run ANN, no seasons
-        # So, reduce the ANN count by the number of times these appear
-        season_counter["ANN"] -= set_counter["enso_diags"]
-        season_counter["ANN"] -= set_counter["streamflow"]
-        if not all(season_counter["ANN"] == count for count in season_counter.values()):
-            self.fail(
-                "In .cfg files, at least one season does not match the count for ANN: {}".format(
-                    season_counter
-                )
-            )
-
```

Comment:
Default diags are no longer ran in this test, so this section will break. I removed this section because I am pretty sure it is not really a necessary test.